bin/run_mne_bem.py

```diff
@@ -0,0 +1,109 @@
+
+
+def _run(subjects_dir, subject, layers, ico, overwrite):
+    this_env = copy.copy(os.environ)
+    this_env['SUBJECTS_DIR'] = subjects_dir
+    this_env['SUBJECT'] = subject
+
+    if 'SUBJECTS_DIR' not in this_env:
+        raise RuntimeError('The environment variable SUBJECTS_DIR should '
+                           'be set')
+
+    if not op.isdir(subjects_dir):
+        raise RuntimeError('subjects directory %s not found, specify using '
+                           'the environment variable SUBJECTS_DIR or '
+                           'the command line option --subjects-dir')
+
+    if 'FREESURFER_HOME' not in this_env:
+        raise RuntimeError('The FreeSurfer environment needs to be set up '
+                           'for this script')
+
+    subj_path = op.join(subjects_dir, subject)
+    if not op.exists(subj_path):
+        raise RuntimeError('%s does not exits. Please check your subject '
+                           'directory path.' % subj_path)
+    
+    logger.info('1. Setting up MRI files...')
+    if overwrite:
+        run_subprocess(['mne_setup_mri', '--mri', 'T1', '--subject', subject, '--overwrite'], env=this_env)
+    else:
+        run_subprocess(['mne_setup_mri', '--mri', 'T1', '--subject', subject], env=this_env)
+
+    logger.info('2. Setting up %d layer BEM...' % layers)
+    if layers == 3:
+        flash05 = op.join(subjects_dir, subject, 'nii/FLASH5.nii')
+        flash30 = op.join(subjects_dir, subject, 'nii/FLASH30.nii')
+
+        run_subprocess(['mne', 'flash_bem_model', '-s', subject, '-d', subjects_dir,
+                        '--flash05', flash05, '--flash30', flash30, '-v'], env=this_env)
+        for srf in ('inner_skull', 'outer_skull', 'outer_skin'):
+            shutil.copy(op.join(subjects_dir, subject, 'bem/flash/%s.surf' % srf),
+                        op.join(subjects_dir, subject, 'bem/%s.surf' % srf))
+    else:
+        if overwrite:
+            run_subprocess(['mne', 'watershed_bem', '-s', subject, '-d', subjects_dir, '--overwrite'], env=this_env)
+        else:
+            run_subprocess(['mne', 'watershed_bem', '-s', subject, '-d', subjects_dir], env=this_env)
+
+    # Create dense head surface and symbolic link to head.fif file
+    logger.info('3. Creating high resolution skin surface for coregisteration...')
+    run_subprocess(['mne', 'make_scalp_surfaces', '--overwrite', '--subject', subject])
+    if op.isfile(op.join(subjects_dir, subject, 'bem/%s-head.fif' % subject)):
+        os.rename(op.join(subjects_dir, subject, 'bem/%s-head.fif' % subject),
+                  op.join(subjects_dir, subject, 'bem/%s-head-sparse.fif' % subject))
+    os.symlink((op.join(subjects_dir, subject, 'bem/%s-head-dense.fif' % subject)),
+               (op.join(subjects_dir, subject, 'bem/%s-head.fif' % subject)))
+
+    # Create source space
+    run_subprocess(['mne_setup_source_space', '--subject', subject, '--spacing', '%.0f' % 5, '--cps'],
+                   env=this_env)
```

Comment:
What source space would you recommend? I've been using a 5mm spaced source space grid for my MEEG study and my fsaverage src space is a 5p7...